doe_dap_dl/dap.py

```diff
@@ -276,14 +281,17 @@ def __renew_cert(self):
         resp = requests.put(f"{self._api_url}/creds", params=params)
 
         if resp.status_code != 200:
-            raise BadStatusCodeError(resp)
-
-        return "cert" in resp.json()
+            if not quiet:
+                self.__print(f"Request to renew certificate returned bad status code {resp.status_code}")
+            return False
 
-    def __cert_is_valid(self):
-        try:
-            return self.__renew_cert()
-        except (ValueError, BadStatusCodeError) as e:
+        if "cert" in resp.json():
+            if not quiet:
```

Comment:
Is `quiet` a variable in this function? I don't see it...

Reply:
It is, you can see the function definition on line 266. I added it because when creating a new certificate, this renew function is called twice. Once to see if a valid certificate already exists, and again after it requests a new certificate to check if it's valid (there may be a better way to do that, but renewing a certificate is how the original version validated it).

So I thought it'd be good to suppress those messages when the renew function is called from within the module.

Reply:
You're absolutely right. I was seeing only the "removed" function definition and got confused. Thanks.